examples/configuration/default.json

```diff
@@ -15,7 +15,7 @@
  "log_configuration": false,
  "compression_disabled": false,
  "memory_limit_disabled": false,
- "error_stack_trace_enabled": false,
+ "error_stack_trace_disabled": false,
```

Comment:
so we change the default?

Reply:
this is just an example config but ya I should flip it.

Reply:
unless I should enable it by default. do you think we should enable the error stack by default if rosetta-cli fails?

Reply:
keep it as is so bu default we spit out error message with stack